src/devices/Common/Iot/Device/Common/GeographicPosition.cs

```diff
@@ -132,6 +132,37 @@ public static void GetDegreesMinutesSeconds(double angle, int secDigits, out dou
             }
         }
 
+        /// <summary>
+        /// Returns the given angle as degree and minutes
+        /// </summary>
+        /// <param name="angle">Input angle, in degrees</param>
+        /// <param name="minDigits">Number of digits for the minutes</param>
+        /// <param name="normalizedVal">Normalized angle value (to -180 to 180)</param>
+        /// <param name="degrees">Full degrees</param>
+        /// <param name="minutes">Minutes including requested number of digits</param>
+        public static void GetDegreesMinutes(double angle, int minDigits, out double normalizedVal, out double degrees, out double minutes)
```

Comment:
should degrees be `int`?

Reply:
Good catch. Double doesn't make sense there.